styles/styles.css

```diff
@@ -0,0 +1,270 @@
+  font-weight: bold;
+  background-color: rgb(116, 171, 42);
+  padding: .01rem 2rem;
+  border-radius: .1rem;
+}
+.checkout-link {
+  text-decoration: none;
+}
+.cart {
+  width: 15%;
+  padding-right: .5rem;
+}
+.cart-cost {
+  display: flex;
+  justify-content: flex-end;
+}
+.cart-cost-button {
+  padding: 3rem 6rem 0rem 0rem;
+  display: flex;
+  flex-direction: column;
+  align-items: flex-end;
+}
+.top-nav {
+  background-color: rgb(231, 231, 231);
+  width: 100%;
+  padding: .5rem 0rem .5rem 12rem;
+}
+.top-nav a {
+  text-decoration: none;
+  color: rgb(11, 38, 66);
+}
+li {
+  display: inline;
+  text-decoration: none;
+  font-weight: bold;
+}
+li::after {
+  content: "/";
+  padding: .8rem;
+}
+li:last-of-type:after {
+  content: none;
+}
+ul {
+  padding-left: 0;
+}
+.header-flexbox {
+  display: flex;
+  justify-content: space-between;
+}
+.logo-tagline {
+  width: 40%;
+}
+.footer-logo {
+  width: 25%;
+}
+h2 {
+  padding: 1rem 0rem;
+}
+.h2-padding {
+  padding: 1rem 0rem 1rem 12rem;
+}
+.dogs-main {
+  width: 100%;
+  min-width: 320px;
+  display: flex;
+  align-items: center;
+  padding: 2rem 12rem 3rem;
+}
+.first-main-image {
+  padding-right: 2.5rem;
+  width: 40%;
+}
+.home-dogs {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+  padding: 0rem 9.625rem 2rem;
+}
+.dog-card {
+  min-width: 180px;
+  max-width: 20%;
+  border: 2px solid rgb(242, 242, 242);
+  margin: 1.25rem 1.25rem;
+}
+.dog-img {
+  width: 100%;
+}
+.more-dogs {
+  margin: 0 auto;
+  width: 10rem;
+  text-align: center;
+  padding: .01rem;
+  margin-bottom: 3rem;
+  background-color: rgb(230, 230, 230);
+  color: rgb(13, 45, 78);
+  font-weight: bold;
+  border-radius: .1rem;
+  display: block;
+}
+.more-dogs-link {
+  text-decoration: none;
+}
+.more-dogs-link:visited {
+  color: rgb(13, 45, 78);
+}
+.adopt {
+  margin: 0 auto;
+  display: block;
+  font-weight: bold;
+  width: 100%;
+  padding-top: .5rem;
+  padding-bottom: .5rem;
+  color: white;
+  border: none;
+  background-color: rgb(116, 171, 42);
+  border-radius: .1rem;
+}
+.card-padding {
+  padding: 0 .75rem 1rem .75rem;
+}
+.all-dogs {
+  padding: 2rem 12rem;
+}
+.blog-main {
+  width: 100%;
+  margin: 0 auto;
+  padding: 2rem 12rem 3rem;
+}
+.blog-post {
+  width: 100%;
+  min-width: 320px;
+  display: flex;
+  align-items: center;
+  padding-bottom: 5rem;
+}
+.blog-img {
+  padding-right: 2.5rem;
+  width: 35%;
+}
+.checkout-main {
+  margin: 0 auto;
+  width: 100%;
+  padding: 0rem 12rem 3rem;
+}
+.checkout-content {
+  display: flex;
+  flex-direction: column;
+}
+.dogs-adopted {
+  display: flex;
+  flex-direction: row;
+}
+.adopt-cost {
+  display: flex;
+  flex-direction: row;
+}
+.submit {
+  font-weight: bold;
+  padding: .5rem 3rem;
+  color: white;
+  border: none;
+  background-color: rgb(116, 171, 42);
+  width: 20%;
+}
+.adopt-img {
+  width: 25%;
+  align-self: center;
+  padding-right: 1rem;
+}
+.radio-input {
+  display: flex;
+  flex-direction: row-reverse;
```

Comment:
hmmm...a clever way of showing things, but a curious use of row-reverse? I'm not sold on this, and maybe there's another way?